src/main/java/de/hysky/skyblocker/utils/NEURepoManager.java

```diff
@@ -49,6 +51,20 @@ public static void init() {
                         }))
                 )
         );
+        SkyblockEvents.JOIN.register(NEURepoManager::load);
+    }
+
+    private static void load() {
```

Comment:
Ideally, this could have a better name (and maybe javadoc) or else it's easy to think that this loads the neu repo.